api/api_private_account.go

```diff
@@ -246,6 +246,29 @@ func (s *PrivateAccountAPI) SendTransaction(ctx context.Context, args SendTxArgs
 	return submitTransaction(ctx, s.b, signedTx.Tx)
 }
 
+// SendTransactionAsFeePayer will create a transaction from the given arguments and
+// tries to sign it as a fee payer with the key associated with args.From. If the
+// given passwd isn't able to decrypt the key it fails.
+func (s *PrivateAccountAPI) SendTransactionAsFeePayer(ctx context.Context, args SendTxArgs, passwd string) (common.Hash, error) {
+	if args.Nonce == nil {
```

Comment:
Is this nonce the sender's one?
I think this nonce is not related with feePayer. So we don't need to lock/unlock it.
But Shouldn't the fee payer sign the transaction with nil nonce?

Reply:
Do you assume the sender account and the fee payer account in the same node?

Reply:
If then why don't you make a one sign API to sign both signature once?

Reply:
You are right. The nonce lock is not required. Thank you!